config/policy.go

```diff
@@ -123,20 +130,20 @@ func (cfg *Policy) Validate() error {
 		cfg.FileSystems = defaultFilesystems
 	}
 
-	if cfg.Name == "" {
-		return errored.Errorf("Name is empty for policy")
+	size, err := cfg.CreateOptions.ActualSize()
+	if cfg.Backends.CRUD != "" && (size == 0 || err != nil) {
+		return errored.Errorf("Size set to zero for non-empty CRUD backend %v", cfg.Backends.CRUD).Combine(err)
 	}
 
-	if cfg.Backends.Mount == "" {
-		return errored.Errorf("Mount backend cannot be empty for policy %v", cfg)
+	if err := ValidatePolicy(cfg); err != nil {
+		return errors.ErrJSONValidation.Combine(err)
 	}
 
-	size, err := cfg.CreateOptions.ActualSize()
-	if cfg.Backends.CRUD != "" && (size == 0 || err != nil) {
-		return errored.Errorf("Size set to zero for non-empty CRUD backend %v", cfg.Backends.CRUD).Combine(err)
+	if cfg.Backends.Mount == "" { // backend should be defined and its validated
+		cfg.Backends = defaultDrivers[cfg.Backend]
```

Comment:
Yes. You're right. I did not check existence because cfg.Backend will always contain one of the available drivers (ceph & nfs as of now, other values will not pass the validation ). But I'll check for existence as a safety measure.